src/components/FixtureRow.tsx

```diff
@@ -36,6 +36,8 @@ function FixtureRow({fixture}: FixtureRowProps) {
                 src={fixture.teams.home.logo}
                 alt="Home team logo"
                 style={{maxWidth: '20px'}}
+                width={20}
```

Comment:
Drive-by fix - these should have fixed widths and heights to prevent layout shifts when the images are loading in